tensorflow/lite/interpreter_builder.cc

```diff
@@ -670,7 +671,7 @@ TfLiteStatus InterpreterBuilder::ParseTensors(
 TfLiteStatus InterpreterBuilder::ApplyDelegates(Interpreter* interpreter) {
   // Apply Flex delegate if applicable.
   if (has_flex_op_) {
```

Comment:
called in InterpreterBuilder::operator() (flex delegate set up as part of setting up the interpreter)